src/deepsparse/transformers/pipelines/pipeline.py

```diff
@@ -172,12 +172,11 @@ def create_pipeline_buckets(
         :param kwargs: keyword args for pipeline creation
         :return: A List[Pipeline] objects representing different buckets
         """
-        pipelines = []
-        for seq_len in sorted(sequence_length):
-            curr_pipeline = Pipeline.create(*args, sequence_length=seq_len, **kwargs)
-            pipelines.append(curr_pipeline)
-
-        return pipelines
+        max_seq_length = max(sequence_length)
+        return [
+            Pipeline.create(*args, sequence_length=max_seq_length, **kwargs)
```

Comment:
This is going to make a list of Pipelines that are all the same, rather than compiled for different sequence lengths. We do want to preserve running the inputs in the engine with different padded lengths since this is a performance optimization. Possibly there is something to be done solely to the tokenizers within each pipeline, or during the bucket routing phase.